api/HTMLAreaElement.json

```diff
@@ -205,10 +205,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": false
```

Comment:
`hreflang` on `HTMLAreaElement` is also a mistaken entry which should be removed.

Reply:
Opened #7539 to remove this.